docs/graph_engine/overview.rst

```diff
@@ -0,0 +1,194 @@
+Graph engine overview
+=====================
+
+Graphs for GNN can be stored in different formats and in order to have ability to plug them in the existing
+algorithms we define an API for basic operations in
+`_base.py <https://github.com/microsoft/DeepGNN/blob/main/src/python/deepgnn/graph_engine/_base.py#L30>`_.
+Typically nodes are represented as integers while edges are triple of integers:
+(source node, destination node, edge type). Each node and edge can store multiple features that can be addressed
```

Comment:
NIT multiple feature vectors

Reply:
I just convert the existing docs to rST.